source/game.database.bmx

```diff
@@ -585,6 +606,64 @@ Type TDatabaseLoader
 				personGenPos = expression.Find("%PERSONGENERATOR_")
 			Wend
 		EndIf
+		
+		'replace sport stuff:
+		If expression.Find("%TEAM")>=0
+			expression = expression.Replace("%TEAM1NAMEINITIALS%", "${.self:~qsportteam~q:0:~qteaminitials~q}")
+			expression = expression.Replace("%TEAM2NAMEINITIALS%", "${.self:~qsportteam~q:1:~qteaminitials~q}")
+			expression = expression.Replace("%TEAM1RANK%", "${.self:~qsportteam~q:0:~qleaguerank~q}")
+			expression = expression.Replace("%TEAM2RANK%", "${.self:~qsportteam~q:1:~qleaguerank~q}")
+			expression = expression.Replace("%TEAM1TRAINERSHORT%", "${.self:~qsportteam~q:0:~qtrainer~q:~qlastname~q}")
+			expression = expression.Replace("%TEAM2TRAINERSHORT%", "${.self:~qsportteam~q:1:~qtrainer~q:~qlastname~q}")
+			expression = expression.Replace("%TEAM1TRAINER%", "${.self:~qsportteam~q:0:~qtrainer~q:~qfullname~q}")
+			expression = expression.Replace("%TEAM2TRAINER%", "${.self:~qsportteam~q:1:~qtrainer~q:~qfullname~q}")
+			expression = expression.Replace("%TEAM1STARSHORT%", "${.self:~qsportteam~q:0:~qmember~q:~qforward~q:~qlastname~q}")
+			expression = expression.Replace("%TEAM2STARSHORT%", "${.self:~qsportteam~q:1:~qmember~q:~qforward~q:~qlastname~q}")
+			expression = expression.Replace("%TEAM1STAR%", "${.self:~qsportteam~q:0:~qmember~q:~qforward~q:~qfullname~q}")
+			expression = expression.Replace("%TEAM2STAR%", "${.self:~qsportteam~q:1:~qmember~q:~qforward~q:~qfullname~q}")
+			expression = expression.Replace("%TEAM1KEEPERSHORT%", "${.self:~qsportteam~q:0:~qmember~q:~qkeeper~q:~qlastname~q}")
+			expression = expression.Replace("%TEAM2KEEPERSHORT%", "${.self:~qsportteam~q:1:~qmember~q:~qkeeper~q:~qlastname~q}")
+			expression = expression.Replace("%TEAM1KEEPER%", "${.self:~qsportteam~q:0:~qmember~q:~qkeeper~q:~qfullname~q}")
+			expression = expression.Replace("%TEAM2KEEPER%", "${.self:~qsportteam~q:1:~qmember~q:~qkeeper~q:~qfullname~q}")
+			expression = expression.Replace("%TEAM1NAME%", "${.self:~qsportteam~q:0:~qteamname~q}")
+			expression = expression.Replace("%TEAM2NAME%", "${.self:~qsportteam~q:1:~qteamname~q}")
+			expression = expression.Replace("%TEAM1CITY%", "${.self:~qsportteam~q:0:~qcity~q}")
+			expression = expression.Replace("%TEAM2CITY%", "${.self:~qsportteam~q:1:~qcity~q}")
+			expression = expression.Replace("%TEAM1ARTICLE1% %TEAM1NAME%", "${.self:~qsportteam~q:0:~qteamnamewitharticle~q}")
+			expression = expression.Replace("%TEAM1ARTICLE2% %TEAM1NAME%", "${.self:~qsportteam~q:0:~qteamnamewitharticle~q:2}")
+			expression = expression.Replace("%TEAM2ARTICLE1% %TEAM2NAME%", "${.self:~qsportteam~q:1:~qteamnamewitharticle~q}")
+			expression = expression.Replace("%TEAM2ARTICLE2% %TEAM2NAME%", "${.self:~qsportteam~q:1:~qteamnamewitharticle~q:2}")
+			expression = expression.Replace("%TEAM2ARTICLE2% %TEAM2%", "${.self:~qsportteam~q:1:~qteamnamewitharticle~q:2}")
```

Comment:
gleiche Ersetzung wie Team2Name; gleiches nicht für Team 1?

Reply:
Das kam in den Texten nicht vor. Da hab uch das einfach weggelassen.